modules/adding-to-a-project.adoc

```diff
@@ -7,9 +7,12 @@
 
 .Procedure
 
-. Navigate to *Home* -> *Projects*.
+. Select *Developer* from the context selector at the top of the web console
+navigation menu.
 
-. Select a project.
+. Click *+Add*
 
-. In the upper right-hand corner of the *Project Status* menu, click *Add*, then
-choose from the provided options.
+. At the top of the page, select the name of the project that you want to add to.
+
+. Click on a method for adding to your project (for example, *From Git*,
```

Comment:
s/Click on a method for adding to your project/Click on a method for adding to your project, _and_ then follow the workflow./